searx/static/themes/simple/src/less/search.less

```diff
@@ -380,10 +380,8 @@ html.no-js #clear_search.hide_if_nojs {
 }
 
 .favicon img {
-  height: 1.8rem;
-  width: 1.8rem;
+  height: 1.25rem;
+  width: 1.25rem;
```

Comment:
I like both, but ``1.25rem`` fits the line height much better

Reply:
The purpose of the favicon is to quickly be able to quickly see what you're clicking on. At 1.25rem it becomes much harder to make out the favicon, and even more so on mobile devices.

An issue is that SearXNG doesn't have website 'short-names'. For example, make a search on Google and you'll see how the short name is above the link which balances out the design.

Obviously, I'm no expert on UX/UI design but I think usability should be prioritised. I would be in favour of maybe lowering the size of the favicons just a little if people want to do that but this is just too much in my opinion.

Reply:
Just popping in with my thoughts.

At `1.8rem`, the favicons are far too large and distract from reading the names for the search results; they're meant to be auxillary content, not the main focus.

On the other hand, `1.25rem` *is* a bit small, and could use a slight bump (maybe `1.5rem`?)

Reply:
> An issue is that SearXNG doesn't have website 'short-names'. For example, make a search on Google and you'll see how the short name is above the link which balances out the design.

I don't think a "short-name" is needed in SearXNG / most origin search sites (e.g. ddg) do not have a short name we can fetch in our engines.

Here is how it looks on google (incl. "short-name") ..

![grafik](https://github.com/user-attachments/assets/2e54c319-8484-46c3-adca-8005099b8755)

and here a screenshot from ddg:

![grafik](https://github.com/user-attachments/assets/182df776-e239-4e0e-a6c7-356051d24dde)

Reply:
Current status (1.25rem width/height, 10% rounding):
![2024-10-19-103501](https://github.com/user-attachments/assets/e5597ee0-7f01-4409-a1b4-5a83a59f1a25)

Reply:
Alternatively, there is this (1.5rem width/height, 10% rounding):
![2024-10-19-104427](https://github.com/user-attachments/assets/631d566f-312e-408f-952e-d9c716bf6f5e)

Which one do you all prefer?

Reply:
> The purpose of the favicon is to quickly be able to quickly see what you're clicking on. At 1.25rem it becomes much harder to make out the favicon, and even more so on mobile devices.

> At `1.8rem`, the favicons are far too large  .. On the other hand, `1.25rem` *is* a bit small, and could use a slight bump (maybe `1.5rem`?)

.. both arguments count, its hard for me to decide for one or the other .. (may we see some day a PR that places the favicon left to the title?) .. 

> Which one do you all prefer?

I'm not an UI expert to judge, but since I have to make a decision I will choose the compromise, which seems to be 1.5rem .. hope thats OK for all of you.